workflow/1-generate_mlbf

```diff
@@ -40,10 +40,10 @@ def main():
 
     cmdline = cmdline + args.identifier
 
-    if not args.nodiff and "GOOGLE_APPLICATION_CREDENTIALS" in os.environ:
+    all_runs = workflow.get_run_identifiers(args.filter_bucket)
+    if all_runs and not args.nodiff and "GOOGLE_APPLICATION_CREDENTIALS" in os.environ:
```

Comment:
It appears the `"GOOGLE_APPLICATION_CREDENTIALS" in os.environ` check is relevant for the line now moved before the `if` condition, right? At least I wouldn't know what the code inside the `if` would do with GCP credentials.

Reply:
Geez, I know exactly how I missed that and it's not flattering to me.

I'll refactor it further as soon as I can get to a computer. Thanks again @smarnach